### DISCUSSION TITLE
Execute method not triggerring (ver3.0.7)

### DISCUSSION THREAD

**Discussion by MubarakPatel** [quartznet/quartznet#1171] [Q&A] [answered]
I have created a windows scheduler service whohc constantly running in the beckground, it works fines and take the job from quartz also but after certain time it didnot pickup any job for execution i had to restart the scheduler serive to pick them up to take the next job.

**Comment by lahma** [comment, marked as answer]
Duplicate discussion for https://github.com/quartznet/quartznet/discussions/1172 .
